.drone.yml

```diff
@@ -6,10 +6,35 @@ steps:
   - name: greeting
     image: alpine
     commands:
-      - echo hello
-      - echo world
+      - echo This step is to ensure that the pipeline itself is functioning properly.
+  - name: build
+    image: maven:3-jdk-11
```

Comment:
This version is too generic.  There are many versions of maven and the maintainers of this image are always upgrading it to the latest.  We should pic a specific version of maven and use that locally as well with the build machine.

This line should be:  maven:3.6.3-jdk-11